runtime-sdk/src/modules/core/mod.rs

```diff
@@ -603,7 +603,7 @@ impl<Cfg: Config> Module<Cfg> {
         ctx: &C,
         mut args: types::EstimateGasQuery,
     ) -> Result<u64, Error> {
-        let mut extra_gas = 0;
+        let mut extra_gas = 0u64;
```

Comment:
This change is not needed, I think.
This line could be moved lower.

Reply:
It is needed because the compiler needs type annotations to resolve the right methods.